google-cloud-storage_transfer-v1/samples/Gemfile

```diff
@@ -0,0 +1,26 @@
+# Copyright 2021 Google LLC
```

Comment:
Can you move those samples to the `google-cloud-storage_transfer` gem (instead of `google-cloud-storage_transfer-v1`)? The former is the gem we expect users to install.

Then to adjust the samples for that location:

*   In the Gemfile, list `gem "google-cloud-storage_transfer"` (instead of `-v1`)
*   Requires should be `require "google/cloud/storage_transfer"` (instead of `/v1`)
*   When creating a client, use:
    ```ruby
    client = Google::Cloud::StorageTransfer.storage_transfer_service
    ```
    instead of
    ```ruby
    client = ::Google::Cloud::StorageTransfer::V1::StorageTransferService::Client.new
    ```

Everything else should be the same.